examples/dreamcast/pvr/modifier_volume_zclip/example.c

```diff
@@ -0,0 +1,304 @@
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 1.0f, 1.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 1.0f, 1.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 1.0f, 1.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0xffffffff, 0x00000000},
+        {PVR_CMD_VERTEX, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0xffffffff, 0x00000000},
+    };
+    float vert[8][4] = {
+        {-1.0f, -1.0f, 1.0f, 1.0f},
+        {-1.0f, -1.0f, -1.0f, 1.0f},
+        {1.0f, -1.0f, 1.0f, 1.0f},
+        {1.0f, -1.0f, -1.0f, 1.0f},
+        {-1.0f, 1.0f, 1.0f, 1.0f},
+        {-1.0f, 1.0f, -1.0f, 1.0f},
+        {1.0f, 1.0f, 1.0f, 1.0f},
+        {1.0f, 1.0f, -1.0f, 1.0f},
+    };
+    float transform[8][4];
+    int index[18] = {
+        1, 0, 3, -2,
+        4, 5, 6, -7,
+        0, 4, 2, 6, 3, 7, 1, 5, 0, -4};
+    pvr_poly_cxt_t cxt;
+    pvr_poly_hdr_t hdr;
+    /* Vertex transform */
+    mat_identity();
+    mat_apply((matrix_t *)pvm);
+    mat_translate(1.0f, 2.0f, -1.0f);
+    mat_scale(2.0f, 2.0f, 2.0f);
+    for (int i = 0; i < 8; i++)
+    {
+        float tmp[4] = {vert[i][0], vert[i][1], vert[i][2], vert[i][3]};
+        mat_trans_nodiv(tmp[0], tmp[1], tmp[2], tmp[3]);
+        transform[i][0] = tmp[0] / tmp[3];
+        transform[i][1] = tmp[1] / tmp[3];
+        transform[i][2] = tmp[2] / tmp[3];
+        transform[i][3] = 1.0f;
+    }
+    /* Face set */
+    for (int i = 0; i < 18; i++)
+    {
+        int idx = index[i];
+        if (idx < 0)
+        {
+            poly[i].flags = PVR_CMD_VERTEX_EOL;
+            idx = -idx;
+        }
+        else
+        {
+            poly[i].flags = PVR_CMD_VERTEX;
+        }
+        poly[i].x = transform[idx][0];
+        poly[i].y = transform[idx][1];
+        poly[i].z = transform[idx][2];
+    }
+    pvr_poly_cxt_txr(&cxt, PVR_LIST_OP_POLY, PVR_TXRFMT_RGB565, 256, 256, box_tex, PVR_FILTER_BILINEAR);
+    pvr_poly_compile(&hdr, &cxt);
+    hdr.cmd |= (1 << 7); /* PVR_MODIFIER_CHEAP_SHADOW */
+    pvr_prim(&hdr, sizeof(hdr));
+    pvr_vertex_commit_zclip(poly, 18);
+}
+
+void draw_plane(float *pvm)
+{
+    pvr_poly_cxt_t cxt;
+    pvr_poly_hdr_t hdr;
+    __attribute__((aligned(32))) static pvr_vertex_t poly[4] = {
```

Comment:
```suggestion
    static pvr_vertex_t poly[4] = {
```
I just actually got these aligned to 32-bytes in the master branch for 2.1.0 release.